middleware/message.js

```diff
@@ -0,0 +1,11 @@
+const setMessage = (req, res, next) => {
```

Comment:
Looks like `middleware/message.js` and `message.js` are the same, can probably get rid of the second one.

Reply:
Good day, please check the changes in updated pull request week13 that addressed the changes: deleted unnecessary  "if" function in deleteTask and and deleted the duplicating message.js file.